packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelForm.tsx

```diff
@@ -208,6 +247,11 @@ export default function ColumnModelForm(props: ColumnModelFormProps) {
                 )
               })}
             </Select>
+            {errorsByField['columnType'] && (
+              <FormHelperText color={'error.main'}>
+                {errorsByField['columnType']}
+              </FormHelperText>
+            )}
```

Comment:
The `Select` component doesn't have `error` and `helperText` props like `TextField` does.

The `FormHelperText` component gives us the same effect